proto/engine/v1/execution_engine.proto

```diff
@@ -105,6 +105,45 @@ message ExecutionPayloadHeaderCapella {
 	bytes withdrawals_root  = 15 [(ethereum.eth.ext.ssz_size) = "32"]; // New in Capella.
 }
 
+message ExecutionPayloadDeneb {
```

Comment:
Is there anyway of doing this by just adding an optional field to the Altair one or similar? this method of having a new protobuf per fork is pretty much bug prone. cc @prestonvanloon